meson.build

```diff
@@ -1189,8 +1189,23 @@ else
 endif
 conf.set10('HAVE_LIBFDISK', have)
 
+want_passwdqc = get_option('passwdqc')
 want_pwquality = get_option('pwquality')
-if want_pwquality != 'false' and not skip_deps
+if want_passwdqc == 'true' and want_pwquality == 'true'
+        error('passwdqc and pwquality cannot be requested simultaneously')
+endif
+
+if want_passwdqc != 'false' and want_pwquality != 'true' and not skip_deps
```

Comment:
Please move this block down, so that in case both are 'auto' and both are installed, the pwquality is still chosen, which is the least surprising end result given this is new